Mapify/SceneInitializers/GameContent/WorldMapSetup.cs

```diff
@@ -107,7 +108,15 @@ public static void UpdateMap(Transform transform)
                 GameObject name = Object.Instantiate(namePrefab, names);
                 TMP_Text tmp = name.GetComponent<TMP_Text>();
                 tmp.rectTransform.localPosition = station.YardCenter.position.ToXZ().Scale(0, Maps.LoadedMap.worldSize, -0.175f, 0.175f);
-                tmp.text = station.stationID;
+
+                if (showStationNamesOnMap)
+                {
+                    tmp.text = station.stationName;
+                }
+                else
+                {
+                    tmp.text = station.stationID;
+                }
```

Comment:
Nit: This could be a ternary.

Reply:
I avoid ternaries because they are harder to read with no real benefit. Readability is important, shorter code is not.  
But since this is your project I will change this to match your code style.